tiledb.rb

```diff
@@ -1,23 +1,23 @@
 class Tiledb < Formula
   desc "Storage management library for sparse and dense array data"
   homepage "https://tiledb.com"
-  url "https://github.com/TileDB-Inc/TileDB/archive/refs/tags/2.12.0.tar.gz"
-  sha256 "212e80769199c13956e1eb31b56a510fb46ee56310234481a6aaf4a07acf2efb"
+  url "https://github.com/TileDB-Inc/TileDB/archive/refs/tags/2.13.0.tar.gz"
+  sha256 "bc5aabc8bb2951a3882b660aa694353de7c22c5ba0a7ab12817452f3f6d2311d"
   license "MIT"
 
   if OS.mac? && Hardware::CPU.arm?
-    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.12.0/tiledb-macos-arm64-2.12.0-ac8a0df.tar.gz"
-    sha256 "15fc3efc1f770680ca7f3a2e67f15b5a37a86cdcf2414eb8629d2124e6c70f54"
+    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.13.0/tiledb-macos-arm64-2.13.0-db00e70.tar.gz"
+    sha256 "090552f639235d9220940c7c74a2ea3d5c4c1e351c00b6098d71230451f2ccb8"
   end
 
   if OS.mac? && Hardware::CPU.intel?
-    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.12.0/tiledb-macos-x86_64-2.12.0-ac8a0df.tar.gz"
-    sha256 "0e77650ab86cb65cd04f9f6fcb6d8bfc600fd59c03805c6b739a10e7b05e5821"
+    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.13.0/tiledb-macos-x86_64-2.13.0-db00e70.tar.gz"
+    sha256 "d91cbe1e709d2b8e844dbcb51e29333573accf92b33a0a5a1ce1661c834ced44"
   end
 
   if OS.linux? && Hardware::CPU.intel?
-    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.12.0/tiledb-linux-x86_64-2.12.0-ac8a0df.tar.gz"
-    sha256 "e27a51db6fa19b753cda3c9fca16950fa39126d48687c8f3df23091755df5ea0"
+    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.13.0/tiledb-linux-x86_64-2.13.0-db00e70.tar.gz"
+    sha256 ""
```

Comment:
`324aecb43cbc16ae7bc0cded1b90e632c36d28b26e45f990922516e5af73428c` for Linux sha256?

Reply:
Thanks